packages/help-center/src/hooks/use-get-zendesk-fields-by-flow.ts

```diff
@@ -0,0 +1,21 @@
+import { isDIFMFlow, isHundredYearDomainFlow, isHundredYearPlanFlow } from '@automattic/onboarding';
+
+export function useGetZendeskFieldsByFlow( flowName: string ) {
```

Comment:
This seems like it could be a normal function, or even an object map. 

```js
{
    [DIFM]: 'hello',
    ...
}
```

Reply:
Yeah, I increased the complexity here because in the beginning I had to return multiple things and I thought that everything has to be a hook now, good suggestion!